torch/distributed/checkpoint/state_dict_loader.py

```diff
@@ -65,7 +65,7 @@ def load_state_dict(
         >>> my_model = MyModule()
         >>> optimizer = Adagrad(my_model.parameters())
         >>> model_state_dict = my_model.state_dict()
-        >>> fs_storage_loader = torch.distributed.checkpoint.FileSystemLoader("/checkpoint/1")
+        >>> fs_storage_loader = torch.distributed.checkpoint.FileSystemReader("/checkpoint/1")
```

Comment:
nit: If this is the correct change, should we rename the left-hand-side variable to `fs_system_reader`?